indra/llimage/llimagetga.cpp

```diff
@@ -570,7 +571,7 @@ bool LLImageTGA::decodeColorMap( LLImageRaw* raw_image, bool rle, bool flipped )
 		case 2:	pixel_decoder = &LLImageTGA::decodeColorMapPixel15; break;
 		case 3:	pixel_decoder = &LLImageTGA::decodeColorMapPixel24; break;
 		case 4:	pixel_decoder = &LLImageTGA::decodeColorMapPixel32; break;
-		default: llassert(0); return false;
+		default: LL_WARNS() << "mColorMapBytesPerEntry not in [1..4]" << LL_ENDL; return false;
```

Comment:
This has potential to be spammy thus causing slowdowns, I think it should either be a DEBUG or _ONCE.